engines/testbed/midi.cpp

```diff
@@ -54,7 +54,6 @@ void MidiTests::waitForMusicToPlay(MidiParser *parser) {
 
 	CursorMan.showMouse(true);
 	while (!quitLoop) {
-		g_system->delayMillis(10);
```

Comment:
This one should stay: it avoids polling events too fast.

Reply:
reverted